.github/workflows/build-release-packages.yml

```
@@ -3,14 +3,20 @@ name: Build Release Packages
 on:
   release:
     types: [published]
-
+  pull_request:
```

```suggestion
  push:
```
I just realized that the secrets will not be accessible on a pull request, rather we should do it on a push event